lms/djangoapps/learner_home/test_serializers.py

```diff
@@ -718,7 +718,6 @@ def test_serialize_entitlement(self, isExpired, isEnrolled):
             "isExpired": bool(entitlement.expired_at),
             "expirationDate": expected_expiration_date,
             "uuid": str(entitlement.uuid),
-            "enrollmentUrl": f"/api/entitlements/v1/entitlements/{entitlement.uuid}/enrollments",
```

Comment:
Frontend used `homeUrl` and `progressUrl` but you can remove `enrollmentUrl`.

Reply:
ah dang, dunno why those were in the ticket. oh well, i'll just make the enrollmentURL change by itself